internal/pkg/cli/dependencies/container.go

```diff
@@ -34,6 +34,8 @@ var (
 // Container provides dependencies used only in the CLI + common dependencies.
 type Container interface {
 	dependencies.Common
+	Ctx() context.Context
+	WithCtx(ctx context.Context) Container
```

Comment:
Pridal som metodu `WithCtx`, aby som mohol nastavit do dependencies ctx, ktory dolnil DD.
Z tohto dovodu som presunul `ctx` z `Common` dependencies do` Cli/Api/Test` dependencies.

Je to preto, lebo chcem aby `WithCtx(ctx context.Context)` vratil uz konkretny `Container` pre `CLI/API`, aby sa to nemuselo pretypovat pri pouziti.